ldap_auth_provider.py

```diff
@@ -254,9 +256,7 @@ async def check_3pid_auth(self, medium, address, password):
 
         # Talk to LDAP and check if this email/password combo is correct
         try:
-            server = ldap3.ServerPool(
-                [ldap3.Server(uri, get_info=None) for uri in self.ldap_uris],
-            )
```

Comment:
I note this didn't pass a `tls` parameter before. Was that a bug?

Reply:
(as a general rule I'd prefer to see a separate PR for unrelated changes like this, but in this case it seems small enough that it's ok)

Reply:
I believe this was mentioned in the description as an additional change:

>  As part of this PR ssl certificate validation in `check_3pid_auth` is fixed also.